stranded/1.1.0/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM staphb/samtools:1.16.1
```

Comment:
any reason why this is not at 1.19 (https://github.com/samtools/samtools/releases/) and using a lower version of samtools ?

Reply:
I was just trying to make minimal changes to the 1.0.0 docker. I can bump those changes.